src/schemas/types.json

```diff
@@ -36,9 +36,7 @@
             "enum": [
```

Comment:
Can't we just remove AudioProfile all together? I don't think it is used anymore

Reply:
Entertainment schema uses AudioProfile schema from Types